src/scales/scale.time.js

```diff
@@ -1,7 +1,7 @@
 /* global window: false */
 'use strict';
 
-var moment = require('moment');
+var adapter = require('../core/core.adapters')._date;
```

Comment:
I might call this variable `dates` to make it a bit more fluent when using it. E.g. `dates.startOf` is easier to read then `adapter.startOf`. Also, if we ever end up with a file with multiple types of adapters (e.g. dates, colors, etc.) then we couldn't call each one `adapter` so it's perhaps slightly more future-proof

Reply:
I would go with `dateAdapter`, and maybe only after a second adapter is introduced. But `dates` looks wrong to me.

Reply:
`dateAdapter` seems a bit too long to have to type evertyime. It'd probably be my least favorite of the three options

Reply:
I think the same about `adapter` being confusing in case of multiple types of adapters. However, `dates` or `date` are names too generic and could easily conflict with local variables. `adapter` is like `helpers`, there is very low risk to use these names as local variables. So I would either keep `adapter` until it becomes an issue or use `dateAdapter` (I also don't like it, but at least it would not conflict).